seq2science/rules/configuration_explain.smk

```diff
@@ -73,7 +73,7 @@ else:
                       end=" and finally were tn5 bias shifted by seq2science." if config.get("tn5_shift", 0) > 0 else "."),
         # "samtools_sort": "Bam files were sorted with samtools v@samtools[samtools].",
         "sambamba_sort": "Bam files were sorted with sambamba v@sambamba[sambamba] (https://doi.org/10.1093/bioinformatics/btv098).",
-        "mark_duplicates": "Afterwards, duplicate reads were removed with picard MarkDuplicates v@picard[picard] (http://broadinstitute.github.io/picard).",
+        "mark_duplicates": "Afterwards, duplicate reads were " + ("removed" if "REMOVE_DUPLICATES=true" in config.get("markduplicates", "") else "marked") + " with picard MarkDuplicates v@picard[picard] (http://broadinstitute.github.io/picard).",
```

Comment:
I am not sure, but I think most/some tools will not consider reads marked as duplicates. But from the comments with pictures seems that is not an issue actually :thinking:

Reply:
dupRadar actually needs reads marked

Reply:
yeah makes sense actually :see_no_evil: